backend/app/management/commands/syncdb.py

```diff
@@ -113,16 +120,49 @@ def handle(self, *args, **options):
 
             for row in values_as_a_dict:
                 print(row)
+                # Filter column headers for model fields
+                model_fields = MODEL_NAME_TO_MODEL[model_name]._meta.get_fields()
+                model_field_names = [field.name for field in model_fields]
+                model_kwargs = {}
+                for header in row.keys():
+                    if header in model_field_names or header == 'map_square_number':
+                        # Check if value in column is a number
+                        value = row[header]
+                        if header in ['number', 'map_square_number', 'photographer']:
+                            try:
+                                value = int(value)
+                                if header == 'map_square_number':
+                                    header = 'map_square'
+                            except ValueError:
+                                continue
```

Comment:
In cases like this, let's print to the console a note that the user should go check the value in the spreadsheet, since there's a mistake in the metadata.